### DISCUSSION TITLE
[Feature] Option to backup only on certain wifi netoworks

### DISCUSSION THREAD

**Discussion by jakejones48** [immich-app/immich#5475] [Feature Request]
### The feature

Add an option to specify which wifi networks to use for backups. When connecting to a local only server this would prevent the repeated failed to backup notifications on Android when connected to a different wifi network.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4176
